core/arch/arm/plat-imx/conf.mk

```diff
@@ -88,6 +88,7 @@ else ifneq (,$(filter $(PLATFORM_FLAVOR),$(mx6ull-flavorlist)))
 $(call force,CFG_MX6,y)
 $(call force,CFG_MX6ULL,y)
 $(call force,CFG_TEE_CORE_NB_CORE,1)
+$(call force,CFG_TZC380,y)
```

Comment:
Should be `CFG_TZC380 ?= y` as per commit log description `... by default.`.

Reply:
The `call force` was used to align the `imx6ull` with the rest of `imx6` family.
Giving the security role TZC380 plays in the context of TEE it is preferable to force it to be enabled.